server/controllers/api/videos/index.ts

```diff
@@ -212,7 +212,9 @@ async function addVideoLegacy (req: express.Request, res: express.Response) {
 async function addVideoResumable (req: express.Request, res: express.Response) {
   const videoPhysicalFile = res.locals.videoFileResumable
   const videoInfo = videoPhysicalFile.metadata
-  const files = { image: { path: getResumableUploadPath(videoPhysicalFile.filename) } }
+  const files = { // only used for audio background generation
+    audioBg: { path: getResumableUploadPath(videoPhysicalFile.metadata.audioBg) }
+  }
 
   return addVideo(req, res, { videoPhysicalFile, videoInfo, files })
```

Comment:
May it be more clear if the files property is only set when `videoPhysicalFile.metadata.audioBg` is truthy?

```
  return addVideo(req, res, {
    videoPhysicalFile,
    videoInfo,
    files: videoPhysicalFile.metadata.audioBg ? {
      audioBg: { path: getResumableUploadPath(videoPhysicalFile.metadata.audioBg) }
    } : null,
  })
```

Reply:
@kontrollanten I might be old school, but IMHO comment > code here.

Reply:
I ended up rewriting the initial request, so hopefully this should be more clear anyway 😉